src/Stratis.Bitcoin.Features.Api/ApiFeature.cs

```diff
@@ -117,29 +119,42 @@ public override void Dispose()
         }
     }
 
-    public sealed class ApiFeatureOptions
+    public sealed class ApiSettingsDefaults
     {
+        /// <summary>The default port used by the API when the node runs on the Stratis network.</summary>
+        public string ApiHost { get; set; }
+        public Network Network { get; set; }
+        public bool UseHttps { get; set; }
+        public string HttpsCertificateFilePath { get; set; }
+        public int ApiPort { get; set; }
+        public int KeepAlive { get; set; }
+
+        public ApiSettingsDefaults(Network network)
+        {
+            this.Network = network;
+            this.UseHttps = false;
+            this.HttpsCertificateFilePath = null;
+            this.ApiHost = "http://localhost";
+            this.ApiPort = network.DefaultAPIPort;
```

Comment:
Pass in `ApiPort` into the constructor directly.

Reply:
See above.